soh/src/code/z_en_item00.c

```diff
@@ -397,17 +397,19 @@ void EnItem00_Init(Actor* thisx, PlayState* play) {
             this->scale = 0.01f;
             break;
         case ITEM00_SHIELD_DEKU:
-            this->actor.objBankIndex = Object_GetIndex(&play->objectCtx, OBJECT_GI_SHIELD_1);
-            Actor_SetObjectDependency(play, &this->actor);
+            // this->actor.objBankIndex = Object_GetIndex(&play->objectCtx, OBJECT_GI_SHIELD_1);
+            // Actor_SetObjectDependency(play, &this->actor);
+            this->actor.objBankIndex = 0;
```

Comment:
do we need to remove these for this PR to work?

Reply:
These are required to make like-likes drop their equipment correctly in scenes that normally don't have like-likes for both this and Crowd Control. That said, I should put these behind those CVars until I can guarantee this doesn't spawn items in places they shouldn't. I already found a Deku Shield in a pot in Spirit Temple for example. Then we can remove these permanently in the future PR where we truly remove object dependency and test it thoroughly.

So yeah, I'll do that and add a comment why it needs to be removed as well.